proto/proto/test_context.proto

```diff
@@ -38,7 +38,29 @@ message UploaderMetadata {
   string variant = 4;
 }
 
+message BazelBuildInformation {
+  string label = 1; // test label for the execution 
+  TestBuildResult result = 2; // result of the build
+}
```

Comment:
Can we get this information per test case/suite? Having it top-level isn't enough for us to dedupe the bazel reruns issue we've been having